tests/test_magnetic_fields.py

```diff
@@ -678,6 +678,18 @@ def test_field_line_integrate(self):
         r, z = field_line_integrate(r0, z0, phis, field)
         np.testing.assert_allclose(r[-1], 10, rtol=1e-6, atol=1e-6)
         np.testing.assert_allclose(z[-1], 0.001, rtol=1e-6, atol=1e-6)
+        # test that bounds work correctly, and stop integration when trajectory
+        # hits the bounds
```

Comment:
Maybe make this a new separate unit test instead of adding it onto this existing test?

Reply:
This is duplicated now that you copied it to a new unit test, so you can delete it.